### PR TITLE
Add a license

### PR DESCRIPTION
**Pull request by orta**
Typically all OSS code we ship at Artsy is under the [MIT license](https://choosealicense.com/licenses/mit/). To quote my post on [the Artsy blog](http://artsy.github.io/blog/2015/12/10/License-and-You/):

> All [code licenses] share one important thing:
>
> * YOU MAY USE THIS
> * NO WARRANTY
> * DISCLAIMER OF LIABILITY
>
> Which is basically the crux of how we can all actually use and write OSS without ending up bankrupt. E.g. when a major component of your infrastructure "fails" - it's not the fault of the author. Example: [Shellshock](https://en.wikipedia.org/wiki/Shellshock_%28software_bug%29)) - no-one tried suing the maintainers of Bash.

This is not code however, but data. We aim for non-code work to be licensed under the [Creative Commons license](http://creativecommons.org/licenses/by/4.0)

* [Ash's blog](https://github.com/ashfurrow/blog/blob/master/LICENSE)
* [Design work for CocoaPods](https://github.com/CocoaPods/Design/blob/master/LICENSE)
* [My DNA](https://github.com/orta/dna)

The Creative Commons license acts very similar to the MIT License in that it allows a lot of re-use for adaption or remixing, and limits our liability for other peoples usage with our data. 

As a relevant example for similar projects: the Cooper Hewitt release all their [collection data](https://github.com/cooperhewitt/collection) with an even more permissive license [Creative Commons Zero](http://creativecommons.org/publicdomain/zero/1.0/) and have [a lot of writing](https://github.com/cooperhewitt/collection/wiki/Date-Usage-Guidelines) around how others can re-use this data.

The only difference between the license I added here, and the one used by the Cooper Hewitt is that cc4.0 has specific terminology about databases and can offer some protection if we ever consider there to be a negative re-use of our data (can't see a use case for that, but we're covered by the [suis generis database rights](https://creativecommons.org/share-your-work/licensing-considerations/version4/)

/cc @ystarr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,4 @@ We ask that you give credit to Artsy if using this list in a public-facing forma
 ----
 ### Need a data set to play with? Check out [Artsy's Public API](http://developers.artsy.net/), where you can experiment with The Art Genome Project data applied to public domain artworks and artists.
 
+<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```
Comment (madeleineb) [inline README.md]: Is there a reason to not include more detail re-attribution, @orta? along the lines of

```html
<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br /><span xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Dataset" property="dct:title" rel="dct:type">The Art Genome Project Vocabulary</span> by <a xmlns:cc="http://creativecommons.org/ns#" href="https://artsy.net" property="cc:attributionName" rel="cc:attributionURL">Art.sy, Inc. d/b/a Artsy</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```

**Comment by orta** [comment]
Feedback addressed, as your OOO @madeleineb - I'll remind you when you get back about this

**Comment by orta** [comment]
I think this lives outside of the public API ToS - it's a related work, but it's derived from _our work_ rather than data obtained through the public API. (It is 2 years old ATM)

**Comment by dblock** [comment]
Looks compatible to me with the API TOS and I don't see a problem with having a cc license for a data shop, by IANAL, @ystarr  should (please) make the final call on merging this.

**Comment by ystarr** [comment]
Is this data dump static?

Regardless, I give this a "thumbs up"!

**Comment by orta** [comment]
Yes 👍

**Pull request merged**